docs/guides/workflows_patterns/deployment_google_cloud_composer.rst

```diff
@@ -0,0 +1,155 @@
+                "validations_GCS_store": {
+                    "class_name": "ValidationsStore",
+                    "store_backend": {
+                        "class_name": "TupleGCSStoreBackend",
+                        "project": "REPLACE ME",  # TODO: replace with your value
+                        "bucket": "REPLACE ME",  # TODO: replace with your value
+                        "prefix": "REPLACE ME",  # TODO: replace with your value
+                    },
+                },
+                "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+            },
+            expectations_store_name="expectations_GCS_store",
+            validations_store_name="validations_GCS_store",
+            evaluation_parameter_store_name="evaluation_parameter_store",
+            data_docs_sites={
+                "gs_site": {
+                    "class_name": "SiteBuilder",
+                    "store_backend": {
+                        "class_name": "TupleGCSStoreBackend",
+                        "project": "REPLACE ME",  # TODO: replace with your value
+                        "bucket": "REPLACE ME",  # TODO: replace with your value
+                        "prefix": "REPLACE ME",  # TODO: replace with your value
+                    },
+                    "site_index_builder": {
+                        "class_name": "DefaultSiteIndexBuilder",
+                        "show_cta_footer": True,
+                    },
+                    "show_how_to_buttons": True,
+                }
+            },
+            validation_operators={
+                "action_list_operator": {
+                    "class_name": "ActionListValidationOperator",
+                    "action_list": [
+                        {
+                            "name": "store_validation_result",
+                            "action": {"class_name": "StoreValidationResultAction"},
+                        },
+                        {
+                            "name": "store_evaluation_params",
+                            "action": {"class_name": "StoreEvaluationParametersAction"},
+                        },
+                        {
+                            "name": "update_data_docs",
+                            "action": {"class_name": "UpdateDataDocsAction"},
+                        },
+                    ],
+                }
+            },
+            anonymous_usage_statistics={
+              "enabled": True
+            }
+        )
+
+        context = BaseDataContext(project_config=project_config)
+
+
+4. Create a DAG with validations
+
+    Here we will follow the instructions for `Running a Validation using a PythonOperator <https://docs.greatexpectations.io/en/latest/guides/workflows_patterns/deployment_airflow.html#running-a-validation-using-a-pythonoperator>`_. Note that we will use the data context instantiated in the previous step in place of reading from the filesystem as in the linked example.
```

Comment:
Should there be code here? Or maybe the way this is phrased is just a little misleading, I'd say "In order to run validations with Great Expectations, follow the instructions for etc" that way it's clearer you're sending the reader to the other page.

Reply:
Makes sense! I agree and changed. I think we could add the code, but was trying to not repeat what was at the link.